### ISSUE TITLE
Warnings for nested properties

### ISSUE THREAD

**Issue by claudiopro** [sasstools/sass-lint#352]
This example of nested properties [from the docs](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#nested_properties):

```scss
.funky {
  font: {
    family: fantasy;
    size: 30em;
    weight: bold;
  }
}
```

Produces the following warnings:

```
2:8  warning  Whitespace required before {                         space-before-brace
2:9  warning  Trailing semicolons required                         trailing-semicolon
3:5  warning  Property `family` appears to be spelled incorrectly  no-misspelled-properties
5:5  warning  Property `weight` appears to be spelled incorrectly  no-misspelled-properties
```

**Comment by claudiopro** [comment]
```
$ sass-lint -V
1.3.1
```

**Comment by DanPurdy** [comment]
Confirmed as issue in 1.3.2, we'll look into it, thanks @claudiopro

**Comment by claudiopro** [comment]
@DanPurdy cheers! Let me know how/if I can help

**Comment by DanPurdy** [comment]
@claudiopro 2 of your 3 issues are sorted in 1.3.2 which we just released, the only one still affected here is `no-misspelled-properties` which I was looking into yesterday, still to decide on a good solution for handling nested properties with it but I think it'll either be fixed in 1.3.3 or 1.4.0 which shouldn't be more than a few days / week away.

**Comment by claudiopro** [comment]
That's awesome, thanks for the update!
